### PR TITLE
doc: initial cut at transaction handling section

### PR DESCRIPTION
**Pull request by mhdawson**
Signed-off-by: Michael Dawson <mdawson@devrus.com>

### CODE REVIEW

**Review by ultraJeff — COMMENTED** [review]
Hey Michael!

Just a few spelling and clarity suggestions from me.

File: docs/functional-components/transaction-handling.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+database you are using if you are planning to use that technique.
+
+If you want to read more in depth about these techniques
+and which one you might want to use, 
+[Saga: The new era of transactions in a microservices architecture](https://www.redhat.com/files/summit/session-assets/2019/T42224.pdf)
+covers them in more depth.
+
+To implement 2 phase commit and saga techniques you may also
+need some form of messaging queing as covered
+in the [message queuing](message-queuing.md) section.
```
Comment (roastlechon) [inline docs/functional-components/transaction-handling.md:65]: During the meeting I mis-commented about composition in Kafka, but was remembering Openwhisk supporting orchestrator/composition of serverless functions. https://github.com/apache/openwhisk-composer

Comment (roastlechon) [reply docs/functional-components/transaction-handling.md:65]: Another example with knative https://knative.dev/blog/articles/orchestrating-cloud-events-with/

**Comment by ultraJeff** [comment]
Looks good to me

**Review by jimlindeman — APPROVED** [review]
Looks good to me

**Pull request merged**